proto/node/akash/downtimedetector/v1beta1/query.proto

```
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package akash.downtimedetector.v1beta1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.61.0)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)